### ISSUE TITLE
Localization of the timeline

### ISSUE THREAD

**Issue by christoph-fricke** [holisticon/mutti#2]
- [x] Accept a `lang` attribute, otherwise parse it from the document.
- [ ] Fully support right-to-left layout.
  - [x] Small layout issues in the timeline heading.
  - [ ] Direction of the past and future have to be inverse for rtl, i.e. the future should be left of today.
